templates/quarkus/events/src/main/java/functions/Function.java

```diff
@@ -0,0 +1,23 @@
+package functions;
+
+import io.quarkus.funqy.Context;
+import io.quarkus.funqy.Funq;
+import io.quarkus.funqy.knative.events.CloudEvent;
+
+public class Function {
+
+    @Funq
+    public Object echo(Object input, @Context CloudEvent cloudEvent) {
+        if (cloudEvent != null) {
+            System.out.println(
+                    "CloudEvent{" +
+                            "id='" + cloudEvent.id() + '\'' +
+                            ", specVersion='" + cloudEvent.specVersion() + '\'' +
+                            ", source='" + cloudEvent.source() + '\'' +
+                            ", subject='" + cloudEvent.subject() + '\'' +
+                            '}');
+        }
+        return input;
```

Comment:
I'm not sure what Quarkus is doing with `input` here. What does the object end up being, for example in the testcase?

Reply:
@lance This is simple echo function. Whatever you pass in it should be returned back.

Reply:
In test-case it's `Integer`. At HTTP level -- whatever JSON value (object,array,string,bool,number,null) you send you should get equivalent JSON value back.